src/core/settings.h

```diff
@@ -34,6 +34,7 @@ typedef struct QUIC_SETTINGS_INTERNAL {
             uint64_t PeerUnidiStreamCount                   : 1;
             uint64_t RetryMemoryLimit                       : 1;
             uint64_t LoadBalancingMode                      : 1;
+            uint64_t FixedServerID                          : 1;
```

Comment:
Should this be exposed to the public settings for apps to dynamically set, or should we just leverage the storage (i.e. registry) interface (for which we don't support Linux atm). I'm leaning towards exposing a public setting.